app/components/ui/Legal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import LandingPageCard from './LandingPageCard';
+
+const HOST_QUERY = 'https://askdarcel.org/search?query=';
+const CATEGORIES_LEGAL = [
+  {
+    title: 'Housing',
+    content: 'Eviction defense, housing help, homeownership',
+    query: 'Housing+Law',
+    imgClass: 'lb-housing',
+  },
+  {
+    title: 'Immigration',
+    content: 'Asylum, SIJS, DACA, T-Visas, U-Visas, VAWA',
+    query: 'Immigration',
+    imgClass: 'lb-immigration',
+  },
+  {
+    title: 'Family & Relationships',
+    content: 'Divorce, custory, guardianship, restraining order, T-Visas, etc.',
+    query: 'Family+Law',
+    imgClass: 'lb-family',
+  },
+  {
+    title: 'Criminal & Tickets',
+    content: 'Criminal process questions, quality of life, record expungement',
+    query: 'Criminal',
+    imgClass: 'lb-criminal',
+  },
+  {
+    title: 'Work, Credit, and Consumer',
+    content: 'Credit/consumer, worker\'s rights, work and family',
+    query: 'Workers+consumers',
```

Comment:
ANGOLIA HELP NEEDED: This query does not get the resources for "workers" and "consumers", but instead the resources for "Workers consumers", which is not the same. How do I do that?